### ISSUE TITLE
how can I use AttributeValue for Xamarin.forms.Map.Pins

### ISSUE THREAD

**Issue by SuperRealTeam** [dotnet/MobileBlazorBindings#394]
Hello 

I hope you are doing well , I know that your very busy these days ,but really I need your help in how can I use AttributeValue for Xamarin.forms.Map.Pins and Pins here is List<Xamarin.form.Map.Pin> when I use paramter with List<Pin> it give me specified cast not vaild like in the attached Image.
![Screenshot_1627173962](https://user-images.githubusercontent.com/80648452/126884441-6da638a2-acce-4ed9-899d-d3e423884e65.png)

**Comment by SuperRealTeam** [comment]
I Try To use some Hacks Like serliaze the List to Json then in ViewHandler I deserilaze it and it really done the job until We get Best Solution form your side.

**Comment by Dreamescaper** [comment]
Usually it involves wrapping the object in delegate using AttributeValueHolderFactory (and unwrapping it in Handler).

**Comment by SuperRealTeam** [comment]
Yes, that's the idea ,so I think jsonize any Generic like list<T> or Ienuramble<T> to be string then de serilasize it from strin to json is it the good wrapper or do u think it have another best wrapper for it.

**Comment by Dreamescaper** [comment]
You simply use `AttributeHelper.ObjectToDelegate` (which uses AttributeValueHolderFactory underneath) in component, and  `AttributeHelper.DelegateToObject<List<Xamarin.Forms.Map.Pin>>` in handler. No need to serialize those. 
See examples below (from InputView).

[Component example](https://github.com/dotnet/MobileBlazorBindings/blob/eaee277ee15fd3ef20f465c213bf6cd840023336/src/Microsoft.MobileBlazorBindings/Elements/InputView.generated.cs#L97)
[Handler example](https://github.com/dotnet/MobileBlazorBindings/blob/eaee277ee15fd3ef20f465c213bf6cd840023336/src/Microsoft.MobileBlazorBindings/Elements/Handlers/InputViewHandler.generated.cs#L48)

**Comment by SuperRealTeam** [comment]
Yes that's Great thanks so much
